### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by kaishi05**
Add release notes for 2.3.3

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,27 @@
 # BetoCQ test suite release history
 
+## 2.3.3 (08/27/2024)
+
+## New
+* Add TDLS support check.
```
Comment (xpconanfan) [inline CHANGELOG.md:6]: can we elaborate on the impact of this addition instead of just stating we added a check?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+* Add Aware device capability check in Aware test which is disabled by default.
+* Enables both STREAM and FILE modes in the function test.
```
Comment (xpconanfan) [inline CHANGELOG.md:8]: does this change the quickstart suite?

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,9 +3,9 @@
 ## 2.3.3 (08/27/2024)
 
 ## New
-* Add TDLS support check.
+* Add TDLS support check  and cut the WLAN speed target by half if TDLS is not supported.
```
Comment (xpconanfan) [inline CHANGELOG.md:6]: remove double space

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,9 +3,9 @@
 ## 2.3.3 (08/27/2024)
 
 ## New
```
Comment (xpconanfan) [inline CHANGELOG.md:5]: which of these affect the quickstart suite?

Comment (xpconanfan) [reply CHANGELOG.md:5]: Can we provide this info as well

**Review by xianyuanjia — APPROVED** [review]

**Pull request merged**